From 717e84dfc31283f8f773190e2f79029d1906137c Mon Sep 17 00:00:00 2001
Message-Id: <717e84dfc31283f8f773190e2f79029d1906137c.1349774431.git.minovotn@redhat.com>
In-Reply-To: <f076c6e7e12f7fd3ec2c6b171c4f154b28e2560b.1349774430.git.minovotn@redhat.com>
References: <f076c6e7e12f7fd3ec2c6b171c4f154b28e2560b.1349774430.git.minovotn@redhat.com>
From: Luiz Capitulino <lcapitulino@redhat.com>
Date: Fri, 5 Oct 2012 17:46:17 +0200
Subject: [PATCH 05/21] target-i386: cpu.h: add CPUArchState

RH-Author: Luiz Capitulino <lcapitulino@redhat.com>
Message-id: <1349459193-18226-4-git-send-email-lcapitulino@redhat.com>
Patchwork-id: 42751
O-Subject: [RHEL6.4 qemu-kvm PATCH 03/19] target-i386: cpu.h: add CPUArchState
Bugzilla: 832458
RH-Acked-by: Marcelo Tosatti <mtosatti@redhat.com>
RH-Acked-by: Markus Armbruster <armbru@redhat.com>
RH-Acked-by: Paolo Bonzini <pbonzini@redhat.com>

On upstream, the CPUState macro got renamed to CPUArchState by commit
9349b4f9fda360f3d9adc4cf4443a1a9b429c17e. Doing the same change
downstream is probably not worth it, as it touches too many files.

However, next commits want to use CPUArchState. Add it then to avoid
conflicts.

Signed-off-by: Luiz Capitulino <lcapitulino@redhat.com>
---
 target-i386/cpu.h | 1 +
 1 file changed, 1 insertion(+)

Signed-off-by: Michal Novotny <minovotn@redhat.com>
---
 target-i386/cpu.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/target-i386/cpu.h b/target-i386/cpu.h
index 114afda..95f6011 100644
--- a/target-i386/cpu.h
+++ b/target-i386/cpu.h
@@ -43,6 +43,7 @@
 #endif
 
 #define CPUState struct CPUX86State
+#define CPUArchState struct CPUX86State
 
 #include "cpu-defs.h"
 
-- 
1.7.11.4

